dlr/dlr-native/build.gradle

```diff
@@ -0,0 +1,18 @@
+plugins {
+    id 'maven-publish'
+    id 'signing'
+}
+
+apply from: file("${rootProject.projectDir}/tools/gradle/cpp-formatter.gradle")
+
+task compileJNI {
+    doFirst {
+        if (System.properties['os.name'].toLowerCase(Locale.ROOT).contains("mac")) {
```

Comment:
Only for Mac, not linux?

Reply:
I will first make Mac work then extend to linux and other platform